dev/csharp/getting_started/changetracking.md

```diff
@@ -0,0 +1,34 @@
+# How to track changes
+
+First you need the simplic change tracking nuget package.
+Then choose the poco you want to track. The nuget package contains some attributes
+
+    // Only set it once for each poco because it symbolizes the primary key
+    [TrackingKey] 
+    
+     // This will be responsible for the translation  
+    [ChangeTrackingDisplayName(Key ="YourLocalizationKey")]
+    
+    // Every Property that has this attribute wont be tracked
+    [IgnoreChangeTracking]
+
+Assign the attribute **[TrackingKey]** to your primary property that you use to store the poco into the database
+
+Furthermore every property you want to track and show it with a flexible name (english & german) needs the attribute **[ChangeTrackingDisplayName(Key ="YourTranslationKey")]** 
+
+After that for every property you dont want to track assign this attribute to it 
```

Comment:
`After that, for every property you don't want to track, assign the following attribute`